frontend/src/lib/components/TaxonomicFilter/TaxonomicFilter.scss

```diff
@@ -48,6 +48,18 @@
             color: var(--link);
             background: var(--side);
             border-color: var(--side);
+
+            .posthog-3000 & {
+                color: var(--default);
+                font-weight: 500;
+                &:not(.taxonomic-pill-active) {
```

Comment:
Plan on removing these classes in https://github.com/PostHog/posthog/pull/18684. What design changes are we looking to achieve here?

Reply:
Just styles them in the interim so they're black instead of orange